docs/platform-openrpc.json

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
could it be possible to move this file under a new folder `src/platform/` ? and maybe `src/platform/openrpc.json`.
my understand is that we will have logic in live-common for it so i say let's land src/platform.
if we have doc i even like when it's collocated with the code, but my understanding is that this could even be used with code so i say we should collocate.

but what's your thought @IAmMorrow ?

@IAmMorrow  btw who can review this spec in our side? i say we land it asap and worse case is we make it evolve over time. we can don't need master to be stable if it's a new & unused src/platform.

Reply:
Yep, I'll update.